javaparser-symbol-solver-core/src/main/java/com/github/javaparser/symbolsolver/reflectionmodel/ReflectionClassDeclaration.java

```diff
@@ -154,6 +154,10 @@ public String getQualifiedName() {
                 methods.add(ref.getCorrespondingDeclaration());
             }
         }
+        // when empty there is no sense in trying to find the most applicable
```

Comment:
Yes, that is correct but I wonder if this optimization is useful

Reply:
I was debugging my other resultion issue and came across that part which got entered a lot even if there was an empty list. Maybe its not a big performance gain but it help me with debugging ;)

Reply:
Ok then :) Merging as tests complete